lib/utils/utils.js

```diff
@@ -200,6 +201,17 @@ function isTaggedTemplateExpression(node) {
 }
 
 /**
+ * Check whether or not a node is a a ThisExpression with name of attrs,
+ * which corresponds to a `this.attrs` call.
+ *
+ * @param {ASTNode} node The node to check.
+ * @returns {boolean} Whether or not the node is a ThisExpression with name of attrs.
+ */
+function isThisAttrsExpression(node) {
```

Comment:
Hm, that's a good point actually. I moved it to the `utils` in order to write tests for this. On the other hand it's true that it might not be used outside of this rule. I can move it to the rule I think.